### ISSUE TITLE
Ability to display already generated diagrams

### ISSUE THREAD

**Issue by jstoiko** [aml-org/aml2html#6]
- [ ] already-generated diagrams will be in the form of SVG files and should follow a similar naming convention as the html/mardown pages, e.g. `schema_<dialectSlug>_<schemaName>.svg`

~~- [ ] illustrate relationships between all Dialects and Schemas on an index.html page~~
~~- [ ] illustrate the relationships between all the schemas used in a given Dialect~~

~~Note: look into https://bramp.github.io/js-sequence-diagrams/~~

**Comment by postatum** [comment]
@jstoiko please provide more details. 
1. Will these diagrams be generated by a separate command?
2. Can't they be generated by existing command and placed directly in templates?
3. How exactly all this should work? What's the input, output?

**Comment by jstoiko** [comment]
Yes, that's the idea. There will be a separate command to generate those diagrams, say `aml2svg`. It may be ran before running `aml2html`, `aml2html` needs to provide a way to add those diagrams to the templates.

**Comment by mattdfuchs** [comment]
I've added `aml2uml` repo to mulesoft-labs which can regenerate diagrams.

**Comment by postatum** [comment]
> I've added `aml2uml` repo to mulesoft-labs which can regenerate diagrams.

I get 404 at https://github.com/mulesoft-labs/aml2uml

**Comment by mattdfuchs** [comment]
Will move at Jonathan's suggestion tomorrow

**Comment by jstoiko** [comment]
@postatum: moved - https://github.com/aml-org/aml2uml

**Comment by postatum** [comment]
> @postatum: moved - https://github.com/aml-org/aml2uml

Good, thanks. Few more questions:
1. Should I just provide a way to display svgs generated somewhere else or should aml2html also use aml2uml to generate them?
2. Will the diagrams be generated for dialects, schemas or both?
3. Where the diagrams should be displayed? Separate page, somewhere in a specific place on the page?
